src/test/java/com/wikia/webdriver/testcases/mercurytests/SignupTests.java

```diff
@@ -75,4 +75,17 @@ public void MercurySignupTest_005_signupErrorTooYoungUser() {
           .typeBirthdate("12", "12", "2009").register();
     mobileSignup.verifyBirthdateError();
   }
+
+  @Test(groups = {"MercurySignupTest_006"})
+  @Execute(onWikia = "ja.ja-test")
+  public void MercurySignupTest_006_japaneseUserSignup() {
```

Comment:
All tests in `SignupTests.java` have the same steps except last verification step.
Can we move those steps to separate method and call it in each test?

One more thing: `MercurySignupTest_006_japaneseUserSignup` and `MercurySignupTest_001_successfulSignup` tests have the same `verifyAvatarAfterSignup` method. Can we combine these tests into one and pass different data from TestNG DataProvider?